app.py

```diff
@@ -39,6 +41,8 @@
 api.add_resource(GameGo, '/session/game/go')
 api.add_resource(BankMoney, '/session/game/bank-money')
 api.add_resource(SessionStartRoute, '/session/start')
+api.add_resource(PropertyPurchase,'/session/game/property-purchase')
+api.add_resource(SessionPayRent, '/session/game/pay-rent')

```

Comment:
Verwijder `SessionPayRent`

Je hebt maar 1 patch en 1 post request nodig, alles gaat in `PropertyPurchase`